extensions/amp-animation/0.1/parsers/css-expr-ast.js

```diff
@@ -1381,12 +1381,18 @@ function noCss() {
 }
 
 /**
- * @param {?string} dim
- * @param {!{width: number, height: number}} size
+ * @param {?string} field
+ * @param {!../../../src/layout-rect.LayoutRect} rect
  * @return {number}
  */
-function getDimSide(dim, size) {
-  return dim == 'w' ? size.width : dim == 'h' ? size.height : 0;
+function getRectField(field, rect) {
+  return field == 'w'
+    ? rect.width
+    : field == 'h'
+    ? rect.height
+    : rect[field] != null
+    ? rect[field]
+    : 0;
```

Comment:
Nit: please make this human readable

Reply:
If by this you mean unfolding the ternaries, I can do that. We could also get blabbing on all day about whether ternary sequences are "human-readable" or not though ;)